core/src/main/scala/io/finch/endpoint/multipart.scala

```diff
@@ -22,7 +22,7 @@ private abstract class Attribute[F[_], A](val name: String, val d: DecodeEntity[
   protected def unparsed(errors: NonEmptyList[Throwable]): Rerunnable[Output[F[A]]]
 
   private def multipart(input: Input): Option[FinagleMultipart] =
-    Try(input.request.multipart).toOption.flatten
```

Comment:
I changed this to be consistent with the `FileUpload` implementation below, since I had to touch both anyway.

Reply:
Nice! Thanks a lot!

I figured we have to do some special plumbing to support the new Multipart API (so we don't decode it twice for two separate endpoints, for example). I'd be happy to give it a shot in the upcoming PR. This looks great regardless - will merge when green!

Reply:
Oh, thanks—I didn't look in detail at the change, just made this switch and tests passed, so I figured it was fine. 😄

Reply:
Oh yeah, don't worry, it's more than fine!